src/components/common/DropDownSearch/DropDownSearch.jsx

```diff
@@ -0,0 +1,17 @@
+import { useParams, Link } from "react-router-dom";
+import css from "./search.module.css"
+
+export const DropDownSearch = (props) => {
+    const { categoryTypeId } = useParams()
+  return (
+    <ul className={css.wrapper}>
+      {props.options ? props.options.map((item) => (
+        <li>
+          <Link to={`/${categoryTypeId}/${item.id}`}>
```

Comment:
```suggestion
          <Link to={`/${item.categoryTypeId}/${item.id}`}>
```
на большинства страницах у нас в урле нету categoryTypeId